src/actions/voting.js

```diff
@@ -146,6 +147,7 @@ export const delegatesFetched = ({ activePeer, q, offset, refresh }) =>
         q,
       },
     ).then(({ delegates, totalCount }) => {
+      updateDelegateCache(delegates, activePeer.options.address || activePeer.options.name);
```

Comment:
IMO it would be a better-divided responsibility to define the localStorage key inside `updateDelegateCache`, so here it would be just `updateDelegateCache(delegates, activePeer)`